pkg/routeagent_driver/handlers/ovn/connection.go

```diff
@@ -245,11 +247,15 @@ func discoverKindOvnKubernetesNetwork(ctx context.Context, k8sClientSet clientse
 		return discoverKindOvnDBClusterNetwork(ctx, ovnDBPod, k8sClientSet)
 	}
 
-	ovnPod, err := FindPod(ctx, k8sClientSet, "app=ovnkube-node")
+	ovnPod, err := FindPod(ctx, k8sClientSet, ovnPodLabel)
```

Comment:
We only use the namespace from the ovnPod. So can we use the namespace of ovnDBPod instead?

Reply:
This is for the case where ovnDB pod is nil. if not nil we use discoverKindOvnDBClusterNetwork